code/modules/mob/living/carbon/carbon.dm

```diff
@@ -16,7 +16,7 @@
 
 /mob/living/carbon/Destroy()
 	stomach_contents?.Cut()
-
+	QDEL_NULL(view_change_sources)
```

Comment:
just null the reference to the list. if no one's holding a reference to it, it'll be GCd anyway

Reply:
```suggestion
	view_change_sources = null
```